output/output.go

```diff
@@ -1,24 +1,39 @@
 package output
 
 import (
-	"encoding/json"
 	"fmt"
 
 	"github.com/codegangsta/cli"
 )
 
-// JSON prints results in JSON format.
-func JSON(i interface{}) {
-	j, _ := json.Marshal(i)
-	fmt.Println(string(j))
-}
-
 // Print prints the results of the CLI command.
```

Comment:
The doc on print should list what the function pointer is intended to do as well as a simple example of use.

Example, pulled straight from our slack conversation:

```
// Print prints the results of the CLI command.
// The function passed to the `Print` function returns data to be output
// Print special cases returns of type `map[string]interface{}` and `[]map[string]interface{}`, otherwise it just prints the return from f.
```